code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -265,6 +265,11 @@
 	else
 		..()
 
+/mob/verb/osscheck()
+	set name = "osscheck"
+	set hidden = TRUE
+	alert(usr, "You are running code from an OSS codebase")
+

```

Comment:
The idea is that, by the time we actually need to use it, the vast majority of people will have forgotten about it.